clients/egui/src/account/suggested_docs.rs

```diff
@@ -0,0 +1,131 @@
+use std::sync::Arc;
+
+use eframe::egui;
+
+use crate::model::DocType;
+
+pub struct SuggestedDocs {
+    files: Vec<SuggestedFile>,
```

Comment:
you can prob choose a better name so that you avoid `self.files` which is a bit ambiguous in our project -- `self.recs` for instance is a lot more readable